packages/engine/package.json

```diff
@@ -33,7 +33,7 @@
   ],
   "dependencies": {
     "@cesium/wasm-splats": "^0.1.0-alpha.2",
-    "@spz-loader/core": "0.1.0",
+    "@spz-loader/core": "^0.3.0",
```

Comment:
@keyboardspecialist Should this be "pinned" to a specific version, `0.3.0`, or should should user download newer versions whenever they run `npm install` local?

The caret, `^`, signals to npm to use the later behavior. If we prefer to pin the exact version, the caret should be omitted.

Reply:
We should be using exact version here.